Error when launching the make task

```
> make analyze
ruby page_grabber.rb
WARNING: Nokogiri was built against LibXML version 2.8.0, but has dynamically loaded 2.9.1
page_grabber.rb:44:in `<main>': bad value for range (ArgumentError)
Makefile:2: recipe for target 'fetch' failed
make: *** [fetch] Error 1

~/press-kit master
> ruby -v
ruby 2.1.1p76 (2014-02-24 revision 45161) [x86_64-cygwin]
```

I'll investagate this myself, created the task to document and in case it takes long to fix.

--- a/page_grabber.rb
+++ b/page_grabber.rb
@@ -4,6 +4,8 @@ require 'nokogiri'
 PAGES_DIR      = "data/pages/"
 PER_SECOND     = 15
 
+FileUtils.mkdir_p "data/pages"
+
 
 def most_recent_id
   return @most_recent_id if @most_recent_id
@@ -17,7 +19,7 @@ def latest_stored_id
   Dir["#{PAGES_DIR}*"].map{ |f| f.gsub(PAGES_DIR, "") }
                       .map(&:to_i)
                       .sort
-                      .last
+                      .last || 0
 end
 
 def link(id)

